### ISSUE TITLE
Four serious accessibility violations allowed per page

### ISSUE THREAD

**Issue by ramsharma-prog** [communitiesuk/funding-service-design-frontend#34]
A change in  FS-537 for accessibility tests allows 4 serious accessibility violations per page. 
Jira ticket ref: [fs636](https://digital.dclg.gov.uk/jira/secure/RapidBoard.jspa?rapidView=110&view=planning&selectedIssue=FS-636&issueLimit=100)
